packages/cli/src/lib/implementation/only-plugins.utils.unit.test.ts

```diff
@@ -35,6 +36,9 @@ describe('filterPluginsBySlug', () => {
 });
 
 describe('filterCategoryByPluginSlug', () => {
+  beforeEach(() => {
+    ui().logger.flushLogs();
+  });
```

Comment:
This looks like a perfect candidate for a setup file.

Reply:
I did not do that on purpose. Described in the PR comment.